features/steps/result_api_steps.py

```diff
@@ -104,10 +115,10 @@ def step_impl(context, name):
                 equal_to(name))
 
 
-@then('the analyser version should be "{version}"')
-def step_impl(context, version):
+@then('the analyser version should not be empty')
+def step_impl(context):
     assert_that(context.chosen_result_json['metadata']['analyzer_version'],
-                equal_to(version))
+                not equal_to(0))
```

Comment:
You are checking it here against zero, but the step says it should not be empty. Here and the solver case as well.